analytics/track.go

```diff
@@ -27,8 +30,19 @@ type tracker struct {
 	waitTimeout time.Duration
 }
 
+type noopTracker struct{}
+
+// Enqueue ...
+func (t noopTracker) Enqueue(eventName string, properties ...Properties) {}
+
+// Wait ...
+func (t noopTracker) Wait() {}
+
 // NewDefaultTracker ...
 func NewDefaultTracker(logger log.Logger, properties ...Properties) Tracker {
+	if disabled, err := strconv.ParseBool(os.Getenv(analyticsDisabledEnv)); err == nil && disabled {
```

Comment:
We could simply check the expected value (when disabled) similarly to the [Bitrise CLI's implementation](https://github.com/bitrise-io/bitrise/blob/master/analytics/state_checker.go#L26):
`s.envRepository.Get(DisabledEnvKey) != "true"`

This would simplify the code and also unify the behaviour amond the two components (I'm not sure, but `strconv.ParseBool` might consider "yes" as a true boolean value, while the CLI only accepts "true").